catalog/ui/search-ui/standard/src/main/webapp/js/controllers/cesium.controller.js

```diff
@@ -358,9 +384,12 @@ define(['application',
                 if (this.mapViews) {
                     this.mapViews.destroy();
                 }
+                if (this.clustering) {
```

Comment:
`this.clustering` for the cesium controller was removed.  Is this dead code now?